configuration/scripts/options/set_env.lapack

```diff
@@ -0,0 +1 @@
+setenv ICE_CPPDEFS -DCICE_USE_LAPACK
```

Comment:
I would prefer "USE_LAPACK" as the CPP as this follows the format of some of the latest changes.

Reply:
Good idea. I've changed it (will push later.)